src/views/DashboardOrganization.js

```diff
@@ -120,28 +145,28 @@ const DashboardOrganization = () => {
           [classes.viewExpanded]: isMenuExpanded,
         })}
       >
-        <Container maxWidth="sm">
-          <BalanceDisplayOrganization />
+        <Container
+          className={classes.dashboardOrganizationContainer}
+          maxWidth="sm"
+        >
+          <Box className={classes.balanceContainer}>
+            <BalanceDisplayOrganization />
+          </Box>
           <AppNote />
-          <ActivityStreamWithTabs basePath={DASHBOARD_PATH} />
+          <Grid item xs={12}>
+            <ButtonGroup className={classes.buttonContainer} fullWidth>
+              <Button isOutline isPrimary to={SEARCH_PATH}>
+                {translate('DashboardOrganization.buttonTrustPeople')}
+              </Button>
+              <Button isOutline to={SEND_PATH}>
+                {translate('DashboardOrganization.buttonSendCircles')}
+              </Button>
+            </ButtonGroup>
+          </Grid>
+          <LastInteractions />
+          <NavigationFloating />
         </Container>
       </View>
-      <ButtonAction
-        aria-label="Generate QR"
-        className={clsx(classes.fabQR, {
-          [classes.fabExpanded]: isMenuExpanded,
-        })}
-        component={Link}
-        to={QR_GENERATOR_PATH}
-      >
-        <IconQRLarge fontSize="large" />
```

Comment:
I think it is this part that should stay 
@triaslucia what did you say about the QR code for shared wallets dashboard?

Reply:
Let's leave it for now. It is not scope of this task. Once Lucia finds solution we can implement it